Place homepage intro text alongside logo on large enough screens.

ptheywood opened the pull request:
Reduces the font size on md displays (768 <= width <= 991px) to match vertical height.

Closes #57

mondus commented:
@willfurnass @ptooley Hi Guys, Could you review this before merging? I am keen that we get @pfortiz to give a talk to our RSE community about the contents of his new book.

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+  <font color="black"> <strong> First Steps in Scientific Programming
+  </strong> <br> <br>
+
+I have just published this book in electronic and print formats (iBooks + Amazon).
```
Worth adding links to retailer sites?

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+    <img src="/images/firstSteps.jpg" width="200px">
+  </div>
+
+  <font color="black"> <strong> First Steps in Scientific Programming
```
Redundant para: blog post title is taken from the YAML metadata at the top of this Markdown document

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+<br><br>
+
+<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
+of Automatic Control and Systems Engineering, The University of
```
Add link to ACSE site

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
+of Automatic Control and Systems Engineering, The University of
+Sheffield</b>, mostly involved with the design and implementation of
+the architecture for the Urban Flows Observatory project.
```
Add link to project site and maybe a sentence on what the project's about.

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+
+<br><br>
+
+<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
```
Do you have a profile page / twitter / github acc that you want to link to?

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+series of concepts found in real-life scientific projects, including  concepts of programming, code testing, internal representation,
+rounding errors, tricks of the trade, advice on best practice to
+store data for long-term usage,  and a broad introduction to the
+Unix environment which they will find when coding for HPC and cloud-computing. I do not focus on any individual language but the elements common to all of them. I cover the value of designing the code carefully, planning for future use and scalability, either using flowcharts or just "generic code". I
```
Don't think cloud computing should be hyphenated.

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -0,0 +1,52 @@
+.. title: Book: First steps in Scientific Programming
+.. author: Patricio F. Ortiz
+.. slug: first_steps_in_scientific_programming
+.. date: 2018-03-01 20:59:50 UTC
```
Sorry that it's taken so long to review your blog post!  It might be an idea to update this date to something more recent to ensure that your post is the most recent on our blog.

willfurnass commented:
@pfortiz Thanks for the blog post and sorry that it's taken so long to review it!  I've suggested some minor formatting changes and additional hyperlinks; once you've made the changes I'll get the post published ASAP.

pfortiz commented:
Hi Will,

I've modified the text following your suggestions and committed it to
GitHub . The only point I did not follow was the one regarding a tweeter
account, I don't do social networks (except for linkedIn) :-)
I think in the near future I'll have to update at least my university
web-page.

Best regards,

Patricio


On Thu, Apr 5, 2018 at 2:28 PM Will Furnass <notifications@github.com>
wrote:

> *@willfurnass* requested changes on this pull request.
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179458926>
> :
>
> > +
> +
> +
> +
> +
> +
> +<div style="background:#fcfcef; display: inline-block; font-size:16px; padding: 1%;">
> +
> +  <div style="float:left; margin-left: 1%; padding: 0.25% 0;">
> +    <img src="/images/firstSteps.jpg" width="200px">
> +  </div>
> +
> +  <font color="black"> <strong> First Steps in Scientific Programming
> +  </strong> <br> <br>
> +
> +I have just published this book in electronic and print formats (iBooks + Amazon).
>
> Worth adding links to retailer sites?
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179459045>
> :
>
> > +.. description:
> +.. type: text
> +-->
> +
> +
> +
> +
> +
> +
> +<div style="background:#fcfcef; display: inline-block; font-size:16px; padding: 1%;">
> +
> +  <div style="float:left; margin-left: 1%; padding: 0.25% 0;">
> +    <img src="/images/firstSteps.jpg" width="200px">
> +  </div>
> +
> +  <font color="black"> <strong> First Steps in Scientific Programming
>
> Redundant para: blog post title is taken from the YAML metadata at the top
> of this Markdown document
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179459224>
> :
>
> > +rounding errors, tricks of the trade, advice on best practice to
> +store data for long-term usage,  and a broad introduction to the
> +Unix environment which they will find when coding for HPC and cloud-computing. I do not focus on any individual language but the elements common to all of them. I cover the value of designing the code carefully, planning for future use and scalability, either using flowcharts or just "generic code". I
> +give an overview of the tools available to tackle different problems.
> +Sections as diverse as working with existing code and working with
> +time are covered.  This book is intended to ease the learning curve
> +for those starting, not for the seasoned scientific programmer.
> +
> +<a href="/images/firstSteps4.pdf">
> +Click here for a more detailed description including a table of contents.
> +</a>
> +
> +<br><br>
> +
> +<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
> +of Automatic Control and Systems Engineering, The University of
>
> Add link to ACSE site
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179459326>
> :
>
> > +Unix environment which they will find when coding for HPC and cloud-computing. I do not focus on any individual language but the elements common to all of them. I cover the value of designing the code carefully, planning for future use and scalability, either using flowcharts or just "generic code". I
> +give an overview of the tools available to tackle different problems.
> +Sections as diverse as working with existing code and working with
> +time are covered.  This book is intended to ease the learning curve
> +for those starting, not for the seasoned scientific programmer.
> +
> +<a href="/images/firstSteps4.pdf">
> +Click here for a more detailed description including a table of contents.
> +</a>
> +
> +<br><br>
> +
> +<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
> +of Automatic Control and Systems Engineering, The University of
> +Sheffield</b>, mostly involved with the design and implementation of
> +the architecture for the Urban Flows Observatory project.
>
> Add link to project site and maybe a sentence on what the project's about.
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179459613>
> :
>
> > +series of concepts found in real-life scientific projects, including  concepts of programming, code testing, internal representation,
> +rounding errors, tricks of the trade, advice on best practice to
> +store data for long-term usage,  and a broad introduction to the
> +Unix environment which they will find when coding for HPC and cloud-computing. I do not focus on any individual language but the elements common to all of them. I cover the value of designing the code carefully, planning for future use and scalability, either using flowcharts or just "generic code". I
> +give an overview of the tools available to tackle different problems.
> +Sections as diverse as working with existing code and working with
> +time are covered.  This book is intended to ease the learning curve
> +for those starting, not for the seasoned scientific programmer.
> +
> +<a href="/images/firstSteps4.pdf">
> +Click here for a more detailed description including a table of contents.
> +</a>
> +
> +<br><br>
> +
> +<strong>Patricio F. Ortiz</strong> is an RSE based on the <b>Department
>
> Do you have a profile page / twitter / github acc that you want to link to?
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179459831>
> :
>
> > +
> +<div style="background:#fcfcef; display: inline-block; font-size:16px; padding: 1%;">
> +
> +  <div style="float:left; margin-left: 1%; padding: 0.25% 0;">
> +    <img src="/images/firstSteps.jpg" width="200px">
> +  </div>
> +
> +  <font color="black"> <strong> First Steps in Scientific Programming
> +  </strong> <br> <br>
> +
> +I have just published this book in electronic and print formats (iBooks + Amazon).
> +This work aims at providing science and engineering students/post-docs a
> +series of concepts found in real-life scientific projects, including  concepts of programming, code testing, internal representation,
> +rounding errors, tricks of the trade, advice on best practice to
> +store data for long-term usage,  and a broad introduction to the
> +Unix environment which they will find when coding for HPC and cloud-computing. I do not focus on any individual language but the elements common to all of them. I cover the value of designing the code carefully, planning for future use and scalability, either using flowcharts or just "generic code". I
>
> Don't think cloud computing should be hyphenated.
> ------------------------------
>
> In site/posts/first_steps_in_scientific_programming.md
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#discussion_r179460188>
> :
>
> > @@ -0,0 +1,52 @@
> +<!--
> +.. title: Book: First steps in Scientific Programming
> +.. author: Patricio F. Ortiz
> +.. slug: first_steps_in_scientific_programming
> +.. date: 2018-03-01 20:59:50 UTC
>
> Sorry that it's taken so long to review your blog post! It might be an
> idea to update this date to something more recent to ensure that your post
> is the most recent on our blog.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/pull/62#pullrequestreview-109705485>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AZfM6aObCZon0YGEIj1I5vbfvKAuiycEks5tlhwQgaJpZM4SaB8y>
> .
>

willfurnass commented:
@pfortiz Understood; social media a slippery slope!

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -21,30 +21,49 @@
+environment, from whether variables, air pollution to behaviour of
+construction materials. Its data can potentially be correlated with data
+dealing with human activities and human health issues, and it could involve several disciplines
+in the long run. The <b>Urban flows observatory</b> is mostly financed by EPSRC
```
Capitals for Flows and Observatory?

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -21,30 +21,49 @@
+dealing with human activities and human health issues, and it could involve several disciplines
+in the long run. The <b>Urban flows observatory</b> is mostly financed by EPSRC
+and UKCRIC. Sheffield is one of a handful of cities involved in this
+nationawide effort.
```
Spelling of 'nationwide'

willfurnass commented on site/posts/first_steps_in_scientific_programming.md:
```diff
@@ -21,30 +21,49 @@
+Sheffield</a>, mostly involved with the design and implementation of
+the architecture for the
+<a href="http://urbanflows.org.uk/">Urban Flows Observatory
+project.</a>.  Its objective is to monitor a number of variables in a urban
```
Multiple periods.

ptheywood committed:
Place homepage intro text alongside logo on large enough screens.

Reduces the font size on md displays (768 <= width <= 991px) to match vertical height.

Closes #57
